dkist/io/asdf/resources/schemas/dataset-1.0.0.yaml

```diff
@@ -23,7 +23,9 @@ properties:
     tag: "core/ndarray-1.0.0"
 
   unit:
-    tag: "unit/unit-1.0.0"
+    anyOf:
+      - tag: "unit/unit-1.0.0"
+      - tag: "tag:stsci.edu:asdf/unit/unit-1.0.0"
```

Comment:
Just to check my understanding; this change to the 1.0.0 schema is to allow these files to be correctly read by the new unit machinery if units are present, but to not require that they have it?...

Reply:
This fixes a bug in the 1.0.0 schema where it wouldn't validate the unit. As it's a backwards compatible change we can make it without updating the schema version.